assets/sass/components/idea-hub/_googlesitekit-idea-hub-dashboard-ideas-widget.scss

```diff
@@ -203,6 +203,30 @@
 		}
 	}
 
+	.googlesitekit-idea-hub__loading-notice {
+		align-items: center;
+		display: flex;
+
+		p {
+			color: $c-denim;
+			font-family: Roboto, "Helvetica Neue", Helvetica, Arial, sans-serif;
+
+			font-size: 0.75rem;
+			font-weight: 500;
+			letter-spacing: 0.3px;
+			line-height: 1.33;
+		}
+
+		.googlesitekit-idea-hub__loading-notice__spinner-wrapper {
+			margin-left: 8px;
+
+			svg { // stylelint-disable-line
```

Comment:
No idea what to do here. 

I only added this to override the rule above, but stylelint gives me the same specificity error when I swap the order of the rules

Maybe I should scope that rule more if we don't want to disable stylelint.  I can see a few other disables :smile: